src/index.js

```diff
@@ -46,6 +46,11 @@ class TracePlugin {
       'post:setup': this.postSetup.bind(this),
       'pre:report': this.preReport.bind(this)
     };
+    this.meta = {
+      name: 'trace',
```

Comment:
@kolanos @pselle what do you think about prefixing our own, `iopipe-trace` in this case?

Reply:
++